src/components/featureHighlight/index.js

```diff
@@ -163,7 +171,7 @@ class FeatureHighlight extends BaseComponent {
     AccessibilityInfo.setAccessibilityFocus(reactTag);
   }
 
-  findTargetNode(target) {
+  static findTargetNode(target) {
```

Comment:
Why static?

Reply:
Because we need to use it in `static getDerivedStateFromProps`

Reply:
oh ok